core/src/main/java/com/google/cloud/sql/core/RetryingCallable.java

```diff
@@ -25,59 +24,57 @@
  * The sleep duration is chosen randomly in the range [sleepDuration, sleepDuration * 2] to avoid
  * causing a thundering herd of requests on failure.
  *
+ * <p>exponentialBackoff calculates a duration based on the attempt i.
+ *
+ * <p>The formula is: base * multi^(attempt + 1 + random)
+ *
+ * <p>With base = 200ms and multi = 1.1618, and random = [0.0, 1.0), the backoff values would fall
```

Comment:
nit: value of multi should be 1.618
```suggestion
 * <p>With base = 200ms and multi = 1.618, and random = [0.0, 1.0), the backoff values would fall
```

Reply:
Fixed.